modules/perinatal/carnet-perinatal.routes.ts

```diff
@@ -21,6 +21,8 @@ class CarnetPerinatalResource extends ResourceBase {
                 ]
             };
         },
+        organizacion: MongoQuery.matchString.withField('controles.organizacion.nombre'),
+        profesional: MongoQuery.matchString.withField('controles.profesional.id'),
```

Comment:
Siendo que el id de profesional de control se guarda como tipo ObjectId la menara correcta de matchear sería:
```suggestion
        profesional: {
            field: 'controles.profesional.id',
            fn: (value) => mongoose.Types.ObjectId(value)
        }
```